core/example/solvers/pme.cpp

```diff
@@ -0,0 +1,417 @@
+#include "definitions.h"
+#include "pme.h"
+#include <cmath>
+#include <gsl/gsl_sf_lambert.h>
+#include <Cabana_AoSoA.hpp>
+#include <Cabana_Parallel.hpp>
+#include <Cabana_ExecutionPolicy.hpp>
+#include <atomic>
+#include <Kokkos_Pair.hpp>
+#include <fftw3.h>
+#include <complex>
+#include <vector>
+#include <time.h>
```

Comment:
Can you use `<chrono>` instead for your timing? Also, we may want to start instrumenting code like this with 
[Kokkos profiling tools](https://github.com/kokkos/kokkos-tools)

Reply:
I've switched to using `<chrono>` in pme.cpp, though I now also recall Rene using `Kokkos::Timer` in main.cpp.